### PR TITLE
Karafecho patch 2

### PR DESCRIPTION
**Pull request by karafecho**
Updated search_term and limit parameters in all_feature YAML files. All files passed validation testing.

### CODE REVIEW

File: config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml [hunk 1]
```diff
@@ -549,306 +475,316 @@ patient:
...
+      search_term: ozone exposure
+    type: integer
   AvgDailyOzoneExposure_StudyAvg:
     categories:
     - biolink:ChemicalEntity
     - biolink:EnvironmentalExposure
     maximum: 5
-    minimum: 1
-    type: integer
+    minimum: 10
```
Comment (maximusunc) [inline config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml:643]: Should the name_lookup be 10 and this be 1?

Comment (karafecho) [reply config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml:643]: Fixed and committed to branch.

Comment (karafecho) [reply config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml:643]: I fixed an error related to one of the age variables and committed to branch. However, your Slack message suggests that I misunderstood your comment. Perhaps you can explain? Thanks!

Comment (karafecho) [reply config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml]: Oh, I see. I set the minimum to 1 and kept the maximum at 5. Committed to branch.

**Review by maximusunc — APPROVED** [review]
Looks good from a somewhat cursory glance!

**Pull request merged**